packages/components/atoms/f-form-field/src/components/FormDropdown.vue

```diff
@@ -70,6 +70,8 @@ export default {
 </script>
 
 <style lang="scss" module>
+$form-input-bg--hover                     : darken($color-container-default, $color-hover-01);
```

Comment:
Small one, but if you want to share this across components, you can move the variables out into the [common.scss](https://github.com/justeat/fozzie-components/blob/master/packages/components/atoms/f-form-field/src/assets/scss/common.scss) file and then those get imported into all components to be used.